internal/cloud/run.go

```diff
@@ -409,7 +436,16 @@ func (s *runService) LogTaskStage(ctx context.Context, run *tfe.Run, stage tfe.S
 }
 
 func (s *runService) LogCostEstimation(ctx context.Context, run *tfe.Run) {
-	if run.CostEstimate != nil && run.CostEstimate.Status != tfe.CostEstimateStatus("unreachable") {
+	checkStatus := func(s tfe.CostEstimateStatus) bool {
```

Comment:
Just nit. Maybe we can early return instead of having an anonymous function here? It's very minor and totally up to you, but I feel just like the way you checked `if pcheck.Status == tfe.PolicyPending || pcheck.Status == tfe.PolicyUnreachable` without an anonymous function in the above file, maybe it's more readable that way?

Something like
```
if run.CostEstimate == nil || run.CostEstimate.Status == tfe.CostEstimateStatus("unreachable") || run.CostEstimate.Status == tfe.CostEstimatePending {
  return
}
```

Reply:
Great suggestion, I find that approach more readable as well :)